features/step_definitions/web.rb

```diff
@@ -315,4 +315,17 @@
     })
     step %Q/the step should succeed/
   end 
-end
\ No newline at end of file
+end
+
+Given /^admin enable demo dynamic plugin successfully$/ do
+  ensure_admin_tagged
+  full_version, major, minor = env.get_version(user: user)
+  plugin_image_tag = full_version.gsub('.','')
+  step %Q|I obtain test data file "templates/oc-manifest-demo-plugin.yaml"|
+  step %Q/I replace lines in "oc-manifest-demo-plugin.yaml":/, table(%{
+    | :latest | : #{plugin_image_tag}} |
```

Comment:
An extra white space between `: #` ?

Reply:
you are right, sorry typo

Reply:
updated, thanks @liangxia for your checking